detection-rules/callback_phishing_docusign_comment.yml

```
@@ -20,7 +20,7 @@ source: |
   // Callback Phishing
   and regex.icontains(body.current_thread.text,
                       (
-                        "mcafee|norton|geek.{0,5}squad|paypal|ebay|symantec|best buy|lifelock"
+                        "mcafee|n(o|0)rt(o|0)n|geek.{0,5}squad|paypal|ebay|symantec|best buy|lifel(o|0)ck"
```

use character class instead of groups
```suggestion
                        "mcafee|n[o0]rt[o0]n|geek.{0,5}squad|paypal|ebay|symantec|best buy|lifel[o0]ck"
```

---

Ah thank you, I forget this exists for single characters.